src/combobox/combobox.component.ts

```diff
@@ -379,6 +383,9 @@ export class ComboBox implements OnChanges, AfterViewInit, AfterContentInit {
 				}
 				this.selected.emit(event);
 			});
+			if (!!this.maxLength && !!this.elementRef.nativeElement.querySelector("input")) {
+				this.elementRef.nativeElement.querySelector("input").setAttribute("maxlength", this.maxLength);
```

Comment:
Is there a reason this couldn't be put directly on the `input` in the template? You can set the value to `null` and it won't have a restriction on the `maxlength`. You will probably have to change `[maxlength]` in to `[attr.maxlength]` however and set the type of the input to be something like `@Input() maxLength: number | null = null;`

Reply:
Oh ok, great, I'll make that change now

Reply:
`maxLength: number` should already include `null` ... or we should make it an optional `:?` ... agree that this should be the native `maxlength` - https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/maxlength